### PR TITLE
add low pt photons (10-14 GeV) to miniAOD for 74x

### PR DESCRIPTION
**Pull request by cmkuo**
add photons with pt between 10 and 14 GeV with H/E < 0.15 and charged hadron isolation < 10 GeV

### CODE REVIEW

**Comment by lgray** [comment]
@cmsbuild please test

File: RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
 reducedEgamma = cms.EDProducer("ReducedEGProducer",
-  keepPhotons = cms.string("pt > 14 && hadTowOverEm()<0.15"), #keep in output
-  slimRelinkPhotons = cms.string("pt > 14 && hadTowOverEm()<0.15"), #keep only slimmed SuperCluster plus seed cluster
+  keepPhotons = cms.string("(pt > 14 && hadTowOverEm()<0.15) || (pt>10 && pt<14 && hadTowOverEm()<0.15 && chargedHadronIso()<10)"), #keep in output
+  slimRelinkPhotons = cms.string("(pt > 14 && hadTowOverEm()<0.15) || (pt>10 && pt<14 && hadTowOverEm()<0.15 && chargedHadronIso()<10)"), #keep only slimmed SuperCluster plus seed cluster
```
Comment (gpetruc) [inline RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py]: It should be <= 14, though it doesn't likely make a difference. 
Still, can you do this last change and do git push again on the same branch to update the PR?can

**Comment by lgray** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #10667 8f981eb9e960dbc5fa89a2fb9a99175776d76dc8

Adding low-pT photons to Mini-AOD. There should be no changes except in Mini-AOD. #10666 and #10665 are the 75X and 76X versions of this PR, and they have already been approved by Reco.

The code changes are satisfactory, and Jenkins tests against baseline CMSSW_7_5_X_2015-08-10-2300 show no significant differences, as expected. The addition of the low-pT photons to Mini-AOD was verified for #10665.

**Comment by lgray** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #10667 5d7768d6cf1f8bbadf765c9400f90000bc537b4f

Second approval after code revision.
#10666 and #10665 are the 75X and 76X versions of this PR, and they have already been approved by Reco.

The code change is satisfactory, and Jenkins tests against baseline CMSSW_7_4_X_2015-08-13-1100 show no significant differences, as expected. The addition of the low-pT photons to Mini-AOD was verified for #10665.

**Comment by davidlange6** [comment]
closing PRs included in #10649

**Pull request closed without merging**